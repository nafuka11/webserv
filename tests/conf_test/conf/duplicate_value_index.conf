server {
    index index.html index.htm;
    index index.html;
}