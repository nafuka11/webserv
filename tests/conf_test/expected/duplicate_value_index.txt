webserv: duplicate "index:index.html" value. This value is already set. in ./conf/duplicate_value_index.conf:3
